ui/app/mainui/AppMain.qml

```diff
@@ -200,8 +200,6 @@ Item {
                     contextMenu.hideEmojiPicker = true
                     contextMenu.isRightClickOnImage = true
                     contextMenu.parent = imagePopup.contentItem
-                    contextMenu.setXPosition = function() { return mouse.x + Style.current.smallPadding }
-                    contextMenu.setYPosition = function() { return mouse.y - Style.current.smallPadding }
```

Comment:
Generally we had the rule that context menus that are opened "under" the cursor should be 4px below the cursor and the either to the left or to the right, depending on whether there's no space on the right.

I don't know if this rule was implemented for this context menu already.
Might be a good chance to make it behave this way (if it doesn't) ?

Reply:
Yeah, that functionality used to be there and got removed at some point (don't remember the reasons). If we are to do it that way, it should be done inside the context menu itself, not externally from here